### ISSUE TITLE
Automatic suggestions not shown for some source languages

### ISSUE THREAD

**Issue by tibormarchynzoom** [WeblateOrg/weblate#5669]
**Describe the issue**

We have enabled 5 machine translation services as below and before version 4.5.0 all works good. Now on 4.5.1 we do not see any automatic suggestions for strings in two newly created projects even there is POST in log requesting translations for it, but no exception when DEBUG is enabled. Project and component settings are same compared to projects where machine translation suggestion is working.

```
MT_SERVICES = (
        "weblate.machinery.google.GoogleTranslation",
        "weblate.machinery.microsoft.MicrosoftCognitiveTranslation",
    "weblate.machinery.weblatetm.WeblateTranslation",
    "weblate.memory.machine.WeblateMemory",
    "weblate.machinery.aws.AWSTranslation",
)
```

Log:
```
[pid: 302942|app: 0|req: 312/312] 10.18.255.11 () {54 vars in 1284 bytes} [Mon Mar 15 13:15:41 2021] GET /translate/webui/webui-localization/sk_SK/?&offset=5 => generated 230491 bytes in 254 msecs (HTTP/1.1 200) 8 headers in 557 bytes (1 switches on c
ore 0)
[pid: 302942|app: 0|req: 313/313] 10.18.255.11 () {58 vars in 1308 bytes} [Mon Mar 15 13:15:41 2021] POST /js/translate/google-translate/259326/ => generated 252 bytes in 167 msecs (HTTP/1.1 200) 8 headers in 549 bytes (1 switches on core 0)
[pid: 302942|app: 0|req: 314/314] 10.18.255.11 () {58 vars in 1316 bytes} [Mon Mar 15 13:15:41 2021] POST /js/translate/microsoft-translator/259326/ => generated 260 bytes in 152 msecs (HTTP/1.1 200) 8 headers in 549 bytes (1 switches on core 0)
[pid: 302942|app: 0|req: 315/315] 10.18.255.11 () {58 vars in 1290 bytes} [Mon Mar 15 13:15:41 2021] POST /js/translate/weblate/259326/ => generated 393 bytes in 50 msecs (HTTP/1.1 200) 8 headers in 549 bytes (1 switches on core 0)
[pid: 302942|app: 0|req: 316/316] 10.18.255.11 () {58 vars in 1328 bytes} [Mon Mar 15 13:15:41 2021] POST /js/translate/weblate-translation-memory/259326/ => generated 138 bytes in 336 msecs (HTTP/1.1 200) 8 headers in 549 bytes (1 switches on core 0)
[pid: 302942|app: 0|req: 317/317] 10.18.255.11 () {58 vars in 1282 bytes} [Mon Mar 15 13:15:42 2021] POST /js/translate/aws/259326/ => generated 226 bytes in 109 msecs (HTTP/1.1 200) 8 headers in 549 bytes (1 switches on core 0)
```

**I already tried**

We have upgraded from 4.5.0 to 4.5.1 to see if it's version bug

**To Reproduce the issue**

1. Navigate to any newly created project and component
2. Start translating and switch tab to Automatic suggestions
3. tab content is empty, no suggestions from local translation memory or cloud services

**Expected behavior**

Weblate translation memory or cloud machine translation result should be displayed

**Screenshots**

**Exception traceback**

no exceptions in log when DEBUG enabled

**Server configuration and status**

Weblate installation: PyPI

 * Weblate: 4.5.1
 * Django: 3.1.7
 * siphashc: 1.2
 * translate-toolkit: 3.3.3
 * lxml: 4.4.1
 * Pillow: 6.2.1
 * bleach: 3.1.4
 * python-dateutil: 2.8.1
 * social-auth-core: 4.1.0
 * social-auth-app-django: 4.0.0
 * django-crispy-forms: 1.9.2
 * oauthlib: 3.1.0
 * django-compressor: 2.4
 * djangorestframework: 3.11.0
 * django-filter: 2.4.0
 * django-appconf: 1.0.3
 * user-agents: 2.0
 * filelock: 3.0.12
 * setuptools: 41.2.0
 * jellyfish: 0.7.2
 * openpyxl: 3.0.0
 * celery: 4.4.7
 * kombu: 4.6.11
 * translation-finder: 2.9
 * weblate-language-data: 2021.2
 * html2text: 2019.9.26
 * pycairo: 1.18.1
 * pygobject: 3.34.0
 * diff-match-patch: 20200713
 * requests: 2.22.0
 * django-redis: 4.11.0
 * hiredis: 1.0.1
 * sentry_sdk: 0.14.3
 * Cython: 0.29.16
 * misaka: 2.1.1
 * GitPython: 3.1.0
 * borgbackup: 1.1.10
 * pyparsing: 2.4.7
 * pyahocorasick: 1.4.1
 * Python: 3.6.8
 * Git: 2.27.0
 * psycopg2-binary: 2.8.3
 * phply: 1.2.5
 * chardet: 3.0.4
 * ruamel.yaml: 0.16.5
 * boto3: 1.12.31
 * aeidon: 1.6.0
 * iniparse: 0.5
 * Mercurial: 4.8.2
 * git-svn: 2.27.0
 * git-review: 1.28.0
 * Redis server: 5.0.3
 * PostgreSQL server: 10.15
 * Database backends: django.db.backends.postgresql
 * Cache backends: default:RedisCache, avatar:FileBasedCache
 * Email setup: django.core.mail.backends.smtp.EmailBackend: mail.xxxxxxxxxxxxxxxxx.com
 * OS encoding: filesystem=utf-8, default=utf-8
 * Celery: redis://localhost:6379, redis://localhost:6379, regular
 * Platform: Linux 4.18.0-240.10.1.el8_3.x86_64 (x86_64)

**Weblate deploy checks**

```
System check identified some issues:
WARNINGS:
?: (security.W004) You have not set a value for the SECURE_HSTS_SECONDS setting. If your entire site is served only over SSL, you may want to consider setting a value and enabling HTTP Strict Transport Security. Be sure to read the documentation first; enabling HSTS carelessly can cause serious, irreversible problems.
INFOS:
?: (weblate.I021) Error collection is not set up, it is highly recommended for production use
        HINT: https://docs.weblate.org/en/weblate-4.5.1/admin/install.html#collecting-errors
?: (weblate.I028) Backups are not configured, it is highly recommended for production use
        HINT: https://docs.weblate.org/en/weblate-4.5.1/admin/backup.html
System check identified 3 issues (1 silenced).
```

**Additional context**

**Comment by nijel** [comment]
Do you get any errors in the user interface? Can you check browser developer tools (usually accessible using F12) what response do these requests have?

**Comment by tibormarchynzoom** [comment]
Hi Michal, no, no error in UI, also all network requests in developer console are OK. Only 1 error is in Console as follows
`Manifest: Line: 7, column: 1, Syntax error. - site.webmanifest:7`

The result is just empty page under "automatic suggestions" tab

![weblate_nothing](https://user-images.githubusercontent.com/41163136/111160425-61f78000-859a-11eb-9722-811f7b34b777.PNG)

**Comment by nijel** [comment]
Can you please share content of one of the machine translation requests?

Also, the problem might be using non-standard language in Weblate - are you sure it worked for the same language before?

**Comment by tibormarchynzoom** [comment]
Hi Michal,

aren't they on backend? I see some XHR in developer console, but there is just response content, not request content.
Here is request exported from HAR

`{
        "_initiator": {
          "type": "script",
          "stack": {
            "callFrames": [
              {
                "functionName": "send",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 655,
                "columnNumber": 35
              },
              {
                "functionName": "ajax",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 632,
                "columnNumber": 30
              },
              {
                "functionName": "FullEditor.fetchMachinery",
                "scriptId": "320",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.cc388dc52705.js",
                "lineNumber": 10,
                "columnNumber": 759
              },
              {
                "functionName": "",
                "scriptId": "320",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.cc388dc52705.js",
                "lineNumber": 10,
                "columnNumber": 225
              },
              {
                "functionName": "FullEditor.initMachinery",
                "scriptId": "320",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.cc388dc52705.js",
                "lineNumber": 10,
                "columnNumber": 167
              },
              {
                "functionName": "",
                "scriptId": "320",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.cc388dc52705.js",
                "lineNumber": 10,
                "columnNumber": 5
              },
              {
                "functionName": "dispatch",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 345,
                "columnNumber": 475
              },
              {
                "functionName": "elemData.handle",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 331,
                "columnNumber": 165
              },
              {
                "functionName": "trigger",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 568,
                "columnNumber": 243
              },
              {
                "functionName": "",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 574,
                "columnNumber": 257
              },
              {
                "functionName": "each",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 28,
                "columnNumber": 216
              },
              {
                "functionName": "each",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 19,
                "columnNumber": 191
              },
              {
                "functionName": "trigger",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 574,
                "columnNumber": 228
              },
              {
                "functionName": "Tab.show",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 1160,
                "columnNumber": 6
              },
              {
                "functionName": "",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 1179,
                "columnNumber": 39
              },
              {
                "functionName": "each",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 28,
                "columnNumber": 216
              },
              {
                "functionName": "each",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 19,
                "columnNumber": 191
              },
              {
                "functionName": "Plugin",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 1176,
                "columnNumber": 36
              },
              {
                "functionName": "",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 1795,
                "columnNumber": 306
              },
              {
                "functionName": "mightThrow",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 235,
                "columnNumber": 17
              },
              {
                "functionName": "process",
                "scriptId": "319",
                "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                "lineNumber": 237,
                "columnNumber": 88
              }
            ],
            "parent": {
              "description": "setTimeout",
              "callFrames": [
                {
                  "functionName": "",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 240,
                  "columnNumber": 7
                },
                {
                  "functionName": "fire",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 224,
                  "columnNumber": 230
                },
                {
                  "functionName": "fireWith",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 230,
                  "columnNumber": 180
                },
                {
                  "functionName": "fire",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 231,
                  "columnNumber": 35
                },
                {
                  "functionName": "fire",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 224,
                  "columnNumber": 230
                },
                {
                  "functionName": "fireWith",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 230,
                  "columnNumber": 180
                },
                {
                  "functionName": "ready",
                  "scriptId": "319",
                  "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                  "lineNumber": 248,
                  "columnNumber": 10
                }
              ],
              "parent": {
                "description": "setTimeout",
                "callFrames": [
                  {
                    "functionName": "",
                    "scriptId": "319",
                    "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                    "lineNumber": 249,
                    "columnNumber": 115
                  },
                  {
                    "functionName": "",
                    "scriptId": "319",
                    "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                    "lineNumber": 13,
                    "columnNumber": 26
                  },
                  {
                    "functionName": "",
                    "scriptId": "319",
                    "url": "https://translate.xxxxxxxx.com/static/CACHE/js/output.b1d790eda854.js",
                    "lineNumber": 13,
                    "columnNumber": 45
                  }
                ]
              }
            }
          }
        },
        "_priority": "High",
        "_resourceType": "xhr",
        "cache": {},
        "connection": "111924",
        "pageref": "page_4",
        "request": {
          "method": "POST",
          "url": "https://translate.xxxxxxxx.com/js/translate/aws/259324/",
          "httpVersion": "HTTP/1.1",
          "headers": [
            {
              "name": "Host",
              "value": "translate.xxxxxxxx.com"
            },
            {
              "name": "Connection",
              "value": "keep-alive"
            },
            {
              "name": "Content-Length",
              "value": "84"
            },
            {
              "name": "Accept",
              "value": "application/json, text/javascript, */*; q=0.01"
            },
            {
              "name": "X-Requested-With",
              "value": "XMLHttpRequest"
            },
            {
              "name": "User-Agent",
              "value": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.82 Safari/537.36 Edg/89.0.774.50"
            },
            {
              "name": "Content-Type",
              "value": "application/x-www-form-urlencoded; charset=UTF-8"
            },
            {
              "name": "Origin",
              "value": "https://translate.xxxxxxxx.com"
            },
            {
              "name": "Sec-Fetch-Site",
              "value": "same-origin"
            },
            {
              "name": "Sec-Fetch-Mode",
              "value": "cors"
            },
            {
              "name": "Sec-Fetch-Dest",
              "value": "empty"
            },
            {
              "name": "Referer",
              "value": "https://translate.xxxxxxxx.com/"
            },
            {
              "name": "Accept-Encoding",
              "value": "gzip, deflate, br"
            },
            {
              "name": "Accept-Language",
              "value": "en-US,en;q=0.9,cs;q=0.8"
            },
            {
              "name": "Cookie",
              "value": "_ga=GA1.2.11533670642; _gid=GA1.2.186121349; sessionid=xxxxxxxxxxxxxxxxxxxxxxxxxxx; django_language=en; translate-tab=#machinery"
            }
          ],
          "queryString": [],
          "cookies": [
            {
              "name": "_ga",
              "value": "GA1.2.11570642",
              "expires": null,
              "httpOnly": false,
              "secure": false
            },
            {
              "name": "_gid",
              "value": "GA1.2.1801349",
              "expires": null,
              "httpOnly": false,
              "secure": false
            },
            {
              "name": "sessionid",
              "value": "xxxxxxxxxxxxxxxxxxx",
              "expires": null,
              "httpOnly": false,
              "secure": false
            },
            {
              "name": "django_language",
              "value": "en",
              "expires": null,
              "httpOnly": false,
              "secure": false
            },
            {
              "name": "translate-tab",
              "value": "#machinery",
              "expires": null,
              "httpOnly": false,
              "secure": false
            }
          ],
          "headersSize": 808,
          "bodySize": 84,
          "postData": {
            "mimeType": "application/x-www-form-urlencoded; charset=UTF-8",
            "text": "csrfmiddlewaretoken=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
            "params": [
              {
                "name": "csrfmiddlewaretoken",
                "value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
              }
            ]
          }
        },
        "response": {
          "status": 200,
          "statusText": "OK",
          "httpVersion": "HTTP/1.1",
          "headers": [
            {
              "name": "Server",
              "value": "nginx/1.14.1"
            },
            {
              "name": "Date",
              "value": "Mon, 15 Mar 2021 14:33:03 GMT"
            },
            {
              "name": "Content-Type",
              "value": "application/json"
            },
            {
              "name": "Transfer-Encoding",
              "value": "chunked"
            },
            {
              "name": "Connection",
              "value": "keep-alive"
            },
            {
              "name": "Content-Security-Policy",
              "value": "default-src 'self'; style-src 'unsafe-inline' 'self'; img-src 'self'; script-src 'self'; connect-src 'self'; object-src 'none'; font-src 'self';frame-src 'none'; frame-ancestors 'none';"
            },
            {
              "name": "X-Frame-Options",
              "value": "DENY"
            },
            {
              "name": "Vary",
              "value": "Cookie"
            },
            {
              "name": "X-Content-Type-Options",
              "value": "nosniff"
            },
            {
              "name": "X-XSS-Protection",
              "value": "1; mode=block"
            },
            {
              "name": "Referrer-Policy",
              "value": "same-origin"
            },
            {
              "name": "Set-Cookie",
              "value": "sessionid=xxxxxxxxxxxxxxxxxxx; expires=Mon, 29 Mar 2021 14:33:03 GMT; HttpOnly; Max-Age=1209600; Path=/; SameSite=Lax; Secure"
            }
          ],
          "cookies": [
            {
              "name": "sessionid",
              "value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
              "path": "/",
              "domain": "translate.xxxxxxxx.com",
              "expires": "2021-03-29T14:34:14.233Z",
              "httpOnly": true,
              "secure": true,
              "sameSite": "Lax"
            }
          ],
          "content": {
            "size": 187,
            "mimeType": "application/json",
            "compression": -11,
            "text": "{\"responseStatus\": 200, \"service\": \"AWS\", \"responseDetails\": \"\", \"translations\": [{\"text\": \"Heslo\", \"quality\": 88, \"service\": \"AWS\", \"source\": \"Password\"}], \"lang\": \"sk_SK\", \"dir\": \"ltr\"}"
          },
          "redirectURL": "",
          "headersSize": 659,
          "bodySize": 198,
          "_transferSize": 857,
          "_error": null
        },
        "serverIPAddress": "xx.xx.xx.xx",
        "startedDateTime": "2021-03-15T14:34:14.232Z",
        "time": 372.3410000093281,
        "timings": {
          "blocked": 0.879000013248995,
          "dns": -1,
          "ssl": -1,
          "connect": -1,
          "send": 0.07100000000000001,
          "wait": 371.00600000398606,
          "receive": 0.38499999209307134,
          "_blocked_queueing": 0.42600001324899495
        }
      }`


What you mean non-standard language? but Yes, it is still working in old projects for sk_SK. but language doesn't matter, it's not working in all languages in new affected projects.

But to be sure, we can continue with CZ translations. In 2 new projects I have cs_CZ and Czech languages (each in one project). It  is doing a request (do not see content in developer console, just response) and response is:

New Affected Porject A, lang Czech, word: "Select an action"
`{"responseStatus": 200, "service": "AWS", "responseDetails": "", "translations": [], "lang": "cs", "dir": "ltr"}`

New Affected Project B, lang cs_CZ, word: "Assign all"
`{"responseStatus": 200, "service": "AWS", "responseDetails": "", "translations": [], "lang": "cs_CZ", "dir": "ltr"}`


In working project with lang cs_CZ I see following response
`{"responseStatus": 200, "service": "AWS", "responseDetails": "", "translations": [{"text": "U\u017eivatelsk\u00e9 jm\u00e9no", "quality": 88, "service": "AWS", "source": "Username"}], "lang": "cs_CZ", "dir": "ltr"}`

**Comment by nijel** [comment]
There is no change in the machinery code between 4.5 and 4.5.1. Maybe you also use a different version of the boto3 library and that makes the difference?

**Comment by tibormarchynzoom** [comment]
but we facing issue after upgrade from version before 4.5.... we was running long time on 4.3, recently we have upgraded to 4.5.0 and new projects were created few days ago (already on 4.5) and now we realized that old projects which exists on 4.3 are working fine, just new projects created on 4.5+ are not working and does not show any machine translation suggestion.

Question is, if there is request to AWS, and there is 100% at least to local JS "https://translate.xxxxxxxx.com/js/translate/aws/259324/" why there is not any result.

If you can please tell me how to enable backend logs of those request I can have a look. but even we have switched weblate to DEBUG we didn't see anything more.

**Comment by nijel** [comment]
The backend errors would be logged even without debug mode. There is nothing logged in case the backend doesn't support the locale.

If this is about projects created, then it might be related to language codes - you seem to have disabled SIMPLIFY_LANGUAGES and that will lead to having both `sk` and `sk_SK` locale codes and that might make the difference....

**Comment by tibormarchynzoom** [comment]
but projects which are working correctly using same language codes.. and we never mix `sk` and `sk_SK` in same project.

I could do a screen share to show the behavior, maybe it's just frontend problem.

Also I'm going to upgrade boto3 and other libs for sure

**Comment by tibormarchynzoom** [comment]
updating package didn't help.

is there any way how Weblate requesting translation from i.e. AWS? can I see somwhow the whole request body, url, etc? Frontend POST requests don't contains any payload when calling `https://translate.xxxxxxxx.com/js/translate/aws/259324/` so I want to see if there is really request to AWS and how it looks

**Comment by tibormarchynzoom** [comment]
So I have did just simple tcpdump to see if weblate do request to AWS or Google Cloud and resultes are:
- old projects created before ve upgraded to version 4.5.0 with language code `cs_CZ` or `sk_SK` - I can see outgoing request to AWS and all other cloud services
- new projects created after we upgraded to version 4.50 with language code `cs_CZ` or `sk_SK` - missing requests to AWS or other cloud service

Result is that frontend do a proper POST request to `https://translate.xxxxxxxx.com/js/translate/aws/259324/` with response 200, but Weblate itself does not trigger request to AWS or others. Unfortunately, there is no exception in log even I have setup everything on DEBUG level.

**Comment by tibormarchynzoom** [comment]
So I have found a problem.

Request to AWS or other cloud machine translation services is not fired when Source Language in component is `English (United States)` which we require to have.

**Example why we must have it.**
We have folder which contains all resources in simple format like `Resource_en.properties`,`Resource_cs.properties` and also monolingual base file we use as source `Resource.properties`. All new translations goes to monolingual base.

When I want to create component as below, I get error:
- File format: `Java UTF-8`
- File mask: `somepath/Resource_*.properties`
- Monolingual base: `somepath/Resource.properties`
- Source language: `English`

And source language is problem. I cannot select `English` in this component because I get following error `There is more than one file for English language, please adjust the filemask and use components for translating different resources.` on `Filemask` field. But there is no more than one English translation except monolingual base. So to save this component I must select that Source Language is `English (United States)`. 

So it should be OK to fire AWS or other cloud service request even if language is based on ISO code `en_US`.

**Comment by nijel** [comment]
Thanks for this, there is indeed a problem with such source languages.

**Comment by tibormarchynzoom** [comment]
Hi Michal, thank you a lot, You are great. I have applied fix to 4.5.1 locally and it's working correctly.
